### PR TITLE
Add conda confinement

### PR DESCRIPTION
**Pull request by ingkebil**
How to create a bash-confined conda env.

### CODE REVIEW

**Comment by henrikstranneheim** [comment]
👍

File: conda/confinement.md [hunk 1]
```diff
@@ -0,0 +1,33 @@
+# Confinement
+
+It is possible to truely compartimentalize your conda env with the inclusion of bash functions, bash env variables, and aliases!
+
+## Use case: aliases
+
+So, you want to only have your aliases available in your conda env only?
```
Comment (northwestwitch) [inline conda/confinement.md]: one too many onlys :)

Comment (ingkebil) [reply conda/confinement.md]: haha, fixing :)

**Comment by ingkebil** [comment]
Cool, merging

**Pull request merged**